crates/db_views/src/comment_view.rs

```diff
@@ -600,21 +606,22 @@ mod tests {
 
     let read_comment_views_no_person = CommentQuery::builder()
       .pool(pool)
-      .sort(Some(CommentSortType::Hot))
+      .sort(Some(CommentSortType::Old))
```

Comment:
This was the main change required to get the tests to not be flaky on both `main` and this branch. It seems like the `Hot` ranking doesn't always update in time for the sort order `Hot` to pick the first comment on the post.

Reply:
Yep that's perfectly fine,we should be using New or Old for sorting for the tests, to be predictable.